### DISCUSSION TITLE
Welcome to orca-dclde Discussions!

### DISCUSSION THREAD

**Discussion by scottveirs** [orcasound/orca-dclde#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by JPalmerK** [comment]
Hi! I'm a bioacoustics researcher interested leveraging imperfect data and annotations. I'm hoping to collage some of the amazing datasets available to bring to one of the DCLDE meetings (hopefully 2026).

**Reply by scottveirs** [reply]
Hey Kaitlin,

Thanks for meeting with Holger, and then the HALLO group today. I was excited by some of the ideas you mentioned and added a few of them to a catch-all "General" section in the README:

```Holger thought (via Kaitlin): expand to global killer whale ecotypes?
Holger suggestion of Norwegian KW annotations?
  Scott Q: did anyone ever annotate KW signals in the PALOA archive from Antarctica?
  Scott: Ask re Icelandic recordings/annotations?
  Scott: Reconnect with Dan Olsen et al. re Gulf of Alaska ecotypes?
  Reach out to Russian KW bioacousticians?
Make stronger connections with past WA/BC and compared annotated data sets with past published work? Are we missing any opportunities?
  NRKWs: Miller's localization work? Volker Deecke's work? Christian Bergler's efforts with NRKW calls from OrcaLab?
  SRKWs: Andy Foote's SRKW work? Monika Wieland's undergrad thesis and subsequent work? Candice Emmons? David Bain?
  Bigg's: How many annotations or call type examples does John Ford have?```

**Reply by scottveirs** [reply]
I'm still learning how to use Github effectively... I'm thinking we float ideas in discussions and/or the README first, then maybe move tasks into a Project where we could work from ideation and surveying labeled data prospects to generating initial proposals, including metadata descriptions of the lowest-hanging fruit.
